test/e2e/test_image.py

```diff
@@ -69,6 +69,42 @@ def post_image_with_file_extension_content_type_mismatch(
             )
         return self._post_response_image.json()["id"] if self._post_response_image.status_code == 201 else None
 
+    def post_test_images(self) -> list[dict]:
+        """
+        Posts three images. The first two images have the same entity ID, the last image has a different one.
+
+        :return: List of dictionaries containing the expected image data returned from a get endpoint in
+                 the form of an `ImageMetadataSchema`.
+        """
+        entity_id_a, entity_id_b = (str(ObjectId()) for _ in range(2))
+
+        # First image
+        image_a_id = self.post_image({**IMAGE_POST_METADATA_DATA_ALL_VALUES, "entity_id": entity_id_a}, "image.jpg")
+
+        # Second image
+        image_b_id = self.post_image(
+            {
+                **IMAGE_POST_METADATA_DATA_ALL_VALUES,
+                "entity_id": entity_id_a,
+            },
+            "image.jpg",
+        )
+
+        # Third image
+        image_c_id = self.post_image(
+            {
+                **IMAGE_POST_METADATA_DATA_ALL_VALUES,
+                "entity_id": entity_id_b,
+            },
+            "image.jpg",
+        )
```

Comment:
```suggestion
        # Second image
        image_b_id = self.post_image({**IMAGE_POST_METADATA_DATA_ALL_VALUES, "entity_id": entity_id_a}, "image.jpg")

        # Third image
        image_c_id = self.post_image({**IMAGE_POST_METADATA_DATA_ALL_VALUES, "entity_id": entity_id_b}, "image.jpg")
```
to match `image_a_id`